js/src/tooltip.js

```diff
@@ -352,12 +340,7 @@ class Tooltip extends BaseComponent {
 
     tip.classList.remove(CLASS_NAME_SHOW)
 
-    // If this is a touch-enabled device we remove the extra
-    // empty mouseover listeners we added for iOS support
-    if ('ontouchstart' in document.documentElement) {
-      [].concat(...document.body.children)
-        .forEach(element => EventHandler.off(element, 'mouseover', noop))
-    }
```

Comment:
Now the mouseover listeners are not removed, because the `_enableIosTouchEventsTrick` method only adds the empty mouseover listeners. :thinking:

Reply:
If you compare the changed side by side, you 'll find out that I only cut it and used it for a method to avoid duplication, so I don't think I have messed something. 
With a smaaaall exception:

On line 300 was like below:


```js 

      if ('ontouchstart' in document.documentElement) {
        [].concat(...document.body.children).forEach(element => {
          EventHandler.on(element, 'mouseover', noop())
        })
      }

```
'noop' was called directly

Reply:
If you see line 355-360 in the previous version (event handlers are removed):
```js
// If this is a touch-enabled device we remove the extra
// empty mouseover listeners we added for iOS support
if ('ontouchstart' in document.documentElement) {
  [].concat(...document.body.children)
    .forEach(element => EventHandler.off(element, 'mouseover', noop))
}
```

Reply:
Oooh sorry. I 've just noticed the difference